ultravox/tools/ds_tool/ds_tool.py

```diff
@@ -131,55 +169,58 @@ class DatasetToolArgs:
     )
 
     def __post_init__(self):
-        assert (
-            not self.upload_split or self.dataset_split
-        ), "Must specify dataset_split when using upload_split"
+        if not self.upload_subset:
```

Comment:
```suggestion
        if not self.upload_subset and self.dataset_subset:
```

Reply:
Isn't the two essentially the same, except one is shorter?

Reply:
perhaps; I was suggesting matching the code on lines 174-175, for easier readability.